SECURITY.md

```diff
@@ -2,14 +2,20 @@
 
 ## Supported Versions
 
-| Version | Supported          |
-| ------- | ------------------ |
-| <= 1.7  | :x:                |
-| 1.8.x   | :white_check_mark: |
-| 2.5.x   | :white_check_mark: |
+| Version | Supported | End-of-life date |
+| ------- | --------- | ---------------- |
+| 1.9.x   | ✅ | 2024-12-31 |
```

Comment:
I'd actually love to drop v1.9 support now. How do you feel about that, @neuecc?